django_socio_grpc/utils/registry_singleton.py

```diff
@@ -139,6 +139,13 @@ def register_serializer_as_message_if_not_exist(
 
         self.registered_app[app_name].registered_messages[message_name] = []
 
+        # Add comment for whole message if proto_comment exists in Meta
+        if hasattr(serializer_instance.Meta, "proto_comment") and not is_request:
```

Comment:
No no need for proto comment for request for now ? 
We don't need it in our usage so that okay for me.

Reply:
It looks like a duplicate in xxxxxRequest, I tried to generate without this condition. I guess comment is OK for message or messageResponse object.